dipy/direction/closest_peak_direction_getter.pyx

```diff
@@ -60,14 +63,22 @@ cdef int closest_peak(cnp.ndarray[cnp.float_t, ndim=2] peak_dirs,
 cdef class BasePmfDirectionGetter(DirectionGetter):
     """A base class for dynamic direction getters"""
 
-    def __init__(self, pmf_gen, max_angle, sphere, pmf_threshold=.1, **kwargs):
+    @deprecated_params('max_angle', since='1.14', until='1.15',
+                       alternative='use min_curvature instead.')
+    def __init__(self, pmf_gen, max_angle, sphere, min_curvature=None,
+                 step_size=None, pmf_threshold=.1, **kwargs):
```

Comment:
Do we need to handle the case where both max_angle and min_curvature are provided? At least warn the user that `min_curvature` takes precedence?

Reply:
yes, we should warn the user. I agree